### ISSUE TITLE
NTR: Modifier: sorbitol consumption worsens symptoms (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7858]
New term request Modifier: sorbitol consumption worsens symptoms (brumme_220523181420):

**Comment by pnrobinson** [comment]
@MickeySegal

Sorbitol consumption can cause symptions without any pre-existing condition, see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6693595/

Therefore, I wonder if it is better to model this as an environmental exposure, or is there a particular disease context?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000953
- label: Sorbitol ingestion
- parents: [Nutrition history (HP:4000123)](https://hpo.jax.org/app/browse/term/HP:4000123)
- pmid: 
- def: A history of ingestion of a relatively high amount of foods containing sorbitol.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7858
- synonyms:
